src/rustdoc_schema.graphql

```diff
@@ -724,6 +726,14 @@ type PrimitiveType implements RawType {
   name: String!
 }
 
+"""
+https://docs.rs/rustdoc-types/latest/rustdoc_types/enum.ItemEnum.html#variant.AssocType
+"""
+type AssociatedType implements RawType {
```

Comment:
```suggestion
type AssociatedType {
```
`RawType` is something else actually and it's going to get significantly reworked, it has some issues right now.

Reply:
Also, this feels like it should be `implements Item` and therefore have all the item properties. Otherwise, we won't have access to spans, for example, which will make writing lints difficult.

Reply:
Yeah, when implementing `RawType`, I don't know what was going through my head but definitely not common sense lol. I've removed that implementation, and we now implement Item in c792a1d